### PR TITLE
Updated NC documentation (admin and user guide)

### PR DESCRIPTION
**Pull request by psatyal**
Updated Nextcloud documentation (both admin and user guide), PR round 2.

**Changelog (admin guide):**

- Rewrote a good portion of the text for clarity and consistency with existing documentation
- Modified styling to respect current documentation standards to put titles and actions (titles, fields, buttons) in **bold** and clickpaths in _italic_.
- Added a summary at the top with anchor links to different sections for easier navigation
- Updated most of the visuals on the OpenProject side and one on the Nextcloud side (adding the integration app) with actual screenshots where possible. The Nextcloud ones and some OpenProject ones (to which I don't have access) still need to be updated
- Used h4 for every step of the step-by-step integration guide

**Changelog (user guide):**

- Added a summary at the top with anchor links to the different sections
- Updated wording and formatting
- Updated front matter with a more descriptive description
- Corrected typos (thanks to @judifer for spotting them)

The changes address all issues raised in this work package (and correct other issues too): https://community.openproject.org/projects/gmbh/work_packages/43776/activity 

@marcalcobe Could you review the text and images please, and update the Nextcloud ones where necessary?
@as-op Asking for your review to ensure the links (anchors and links between NC user and admin guides) function properly. Thank you.

### CODE REVIEW

**Review by marcalcobe — APPROVED** [review]
I've double checked the changes on both text and images and it makes sense.

File: docs/user-guide/nextcloud-integration/README.md [hunk 1]
```diff
@@ -1,53 +1,59 @@
...
-## Benefits of the integration
+> Note: The the minimum requirements for this integration are Nextcloud version 22 (or above) and OpenProject version  12.2 (or above). To be able to use Nextcloud as a file storage in your project, the administrator of your instance should first have completed the [Nextcloud integration setup](../../system-admin-guide/integrations/nextcloud).
+
+
+
+| Topic                                                        | Description                                                  |
+| ------------------------------------------------------------ | :----------------------------------------------------------- |
+| [Connecting your OpenProject and Nextcloud accounts](#connecting-your-openproject-and-nextcloud-accounts) | How to connect your Nextcloud and OpenProject accounts to be able to use this integration |
+| [Using the Nextcloud integration](#using-the-Nextcloud-integration) | How to link/unlink files and folders to work packages and view and download linked files |
+| [Next steps for the integration](next-steps-for-the-integration) | What is planned in future releases                           |
```
Comment (as-op) [inline docs/user-guide/nextcloud-integration/README.md]: Anchor links must have the # in front:
#next-steps-for-the-integration

File: docs/user-guide/nextcloud-integration/README.md [hunk 2]
```diff
@@ -1,53 +1,59 @@
...
+| [Possible errors and troubleshooting](possible-errors-and-troubleshooting) | Common errors and how to troubleshoot them                   |
```
Comment (as-op) [inline docs/user-guide/nextcloud-integration/README.md]: Anchor links must have the # in front:
#possible-errors-and-troubleshooting

File: docs/system-admin-guide/integrations/nextcloud/README.md [hunk 3]
```diff
@@ -1,100 +1,183 @@
...
+In the page that follows, under, make sure your newly-created Nextcloud file storage is selected under **Storage**. If you only have one Nextcloud file storage set up (which is the most likely scenario), it should already be selected by default.
 
-If you need to reset the values of the Nextcloud integration (as an administrator), you can always reset the OAuth values from both sides of the integration by clicking on the **Reset OAuth values** (in Nextcloud) or **Replace OAuth values** (in OpenProject) buttons.
+![Add your new file storage to a project](add-file-storage-to-project.png)
+
+Click on **Add** to add your new Nextcloud file storage to this project.
+
+The Nextcloud file storage is now available to all work packages in this project. 
+
+> **Note:** For information on how to use the file storage (link Nextcloud user accounts at a user level, link files to a work package, view and download linked files, unlink files), please read our [Nextcloud integration user guide](../..//user-guide/nextcloud-integration/).
```
Comment (as-op) [inline docs/system-admin-guide/integrations/nextcloud/README.md]: Invalid path to "Nextcloud integration user guide" 
Please change to `../../../user-guide/nextcloud-integration/`

File: docs/system-admin-guide/integrations/nextcloud/README.md [hunk 4]
```diff
@@ -1,100 +1,183 @@
...
+![Nextcloud also generates OAuth values that need to be copied to OpenProject](3_2_04-NC_Step_3.png)
+
+Navigate back to your OpenProject tab and click on the **Done. Continue setup** button on the screen you previously left it at. You will now see a screen where you will be able to enter the Nextcloud values. 
 
 ![OpenProject_NC_OAuth_values](3_3_01-OP_OAuth_application_details.png)
 
-Once these values are entered, you can click the button **Save and complete setup** in your OpenProject tab and the **Yes, I have copied these values** in Nextcloud. Once this is done, your instance configuration is complete.
+Once you hava entered the client ID and client secrets on this page, click on **Save and complete setup**. In the next screen, click on **Yes, I have copied these values**. At this point, your instance configuration is complete and you should see a green banner confirming this.
+
+![Integration successfully completed on the OpenProject end](/Users/parimal/Documents/GitHub/openproject/docs/system-admin-guide/integrations/nextcloud/Nextcloud-set-up-in-OP.png)
```
Comment (as-op) [inline docs/system-admin-guide/integrations/nextcloud/README.md]: Please remove local user path
Change it to: (Nextcloud-set-up-in-OP.png)

**Review by as-op — CHANGES_REQUESTED** [review]
I found some minor issues, please have look.

File: docs/system-admin-guide/integrations/nextcloud/README.md (hunk 4 above)
Comment (psatyal) [reply docs/system-admin-guide/integrations/nextcloud/README.md]: Done.

File: docs/system-admin-guide/integrations/nextcloud/README.md (hunk 3 above)
Comment (psatyal) [reply docs/system-admin-guide/integrations/nextcloud/README.md]: Done.

File: docs/user-guide/nextcloud-integration/README.md (hunk 2 above)
Comment (psatyal) [reply docs/user-guide/nextcloud-integration/README.md]: Done.

File: docs/user-guide/nextcloud-integration/README.md (hunk 1 above)
Comment (psatyal) [reply docs/user-guide/nextcloud-integration/README.md]: Done.

**Comment by psatyal** [comment]
> I found some minor issues, please have look.

Thank you @as-op. Issues fixed.

**Review by psatyal — COMMENTED** [review]
Changes made.

**Review by as-op — APPROVED** [review]

**Pull request merged**